src/ralph/conf.py

```diff
@@ -133,6 +133,36 @@ class Config:  # pylint: disable=missing-class-docstring # noqa: D106
     timeout: float
 
 
+class AuthBackend(Enum):
+    """Model for valid authentication methods."""
+
+    BASIC = "Basic"
+    OIDC = "OIDC"
```

Comment:
Shouldn't all values be upper or lower case but not a mix?

Reply:
all lower then ?